app/src/components/Layers/MapLayers.jsx

```diff
@@ -9,18 +9,29 @@ import GLContainer from 'components/Layers/GLContainer';
 import CustomLayerWrapper from 'components/Layers/CustomLayerWrapper';
 import PolygonReport from 'containers/Map/PolygonReport';
 import ClusterInfoWindow from 'containers/Map/ClusterInfoWindow';
-import { VESSELS_HEATMAP_STYLE_ZOOM_THRESHOLD } from 'config';
+import { VESSELS_RADIAL_GRADIENT_STYLE_ZOOM_THRESHOLD } from 'config';
 import { LAYER_TYPES } from 'constants';
 
-const useHeatmapStyle = zoom => zoom < VESSELS_HEATMAP_STYLE_ZOOM_THRESHOLD;
+const useRadialGradientStyle = zoom => zoom < VESSELS_RADIAL_GRADIENT_STYLE_ZOOM_THRESHOLD;
 
-const getTracks = vessels => vessels
+// TODO this should be in a reducer or container
+// TODO remove vesselTracks in favor of tracks
+const getTracks = (vesselTracks, tracks) => vesselTracks
```

Comment:
not really sure what's the difference between tracks and vesselTracks here. Could we have a clearer naming, or some doc to help?

Reply:
I have added some explanation in the comment. 
I plan to move the tracks management to its own reducer/store, apart from vesselInfo. I did that but only for encounters, so for now the 2 tracks arrays they have to be merged at the component level.
Later I will do a refacto and we will be able to get rid of this, but not in this branch.